app/src/main/java/com/example/homework3/fragments/ListFragment.kt

```diff
@@ -0,0 +1,105 @@
+package com.example.homework3.fragments
+
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.widget.SearchView
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.homework3.adapter.UserAdapter
+import com.example.homework3.databinding.FragmentListBinding
+import com.example.homework3.model.GithubUser
+import com.example.homework3.retrofit.GithubService
+import com.example.homework3.R
+import com.google.android.material.snackbar.Snackbar
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+
+class ListFragment : Fragment() {
+    private var _binding: FragmentListBinding? = null
+    private val binding: FragmentListBinding
+        get() {
+            val requireNotNull = requireNotNull(_binding) {
+                "View was destroyed"
+            }
+            return requireNotNull
+        }
+
+    private val adapter = UserAdapter { user ->
+        findNavController().navigate(
+            com.example.homework3.fragments.ListFragmentDirections.toDetails(
+                user.login
+            )
+        )
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return FragmentListBinding.inflate(inflater, container, false)
+            .also { binding ->
+                _binding = binding
+            }
+            .root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        loadUsers()
+
+        with(binding) {
+
+            toolbar.menu
+                .findItem(R.id.action_search)
+                .let { it.actionView as SearchView }
+                .setOnQueryTextListener(object : SearchView.OnQueryTextListener {
+                    override fun onQueryTextSubmit(query: String): Boolean {
+                        return true
+                    }
+
+                    override fun onQueryTextChange(newText: String): Boolean {
+                        return true
+                    }
+                })
+
+            val linearLayoutManager = LinearLayoutManager(
+                view.context, LinearLayoutManager.VERTICAL, false
+            )
```

Comment:
если посмотреть в дефолный конструктор, то там тоже самое, можно сделать так
```suggestion
            val linearLayoutManager = LinearLayoutManager(view.context)
```